styles/components/_tabbedModal.scss

```diff
@@ -30,6 +30,10 @@
     }
   }
 
+  .tabContent {
```

Comment:
Let's leave it as is. I definitely wouldn't do percentages, since as of now, these modals aren't responsive and there could be a good amount of scope to make them so.